.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
![image](https://github.com/techmoves/todo-list/assets/93039359/b68dddb6-8c3e-4ab7-bafd-88f5228bd2f0)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request more professional 👍